include/wx/window.h

```diff
@@ -1541,6 +1548,15 @@ class WXDLLIMPEXP_CORE wxWindowBase : public wxEvtHandler
     // window is part of a composite control.
     bool WXSendContextMenuEvent(const wxPoint& posInScreenCoords);
 
+    // This internal function needs to be called to set the fitting client size
+    // (i.e. the minimum size determined by the window sizer) when the window
+    // decorations may yet be unknown, as is the case immediately after
```

Comment:
I don't think this issue is related to window decorations. It's caused by GTK not updating cached style information (specifically font sizes, which affect best size calculations) until the TLW is shown.